itou/templates/includes/demo_accounts.html

```diff
@@ -5,26 +5,31 @@
 {% if SHOW_DEMO_ACCOUNTS_BANNER %}
     <div class="alert alert-danger alert-dismissible fade show" role="status">
         <button type="button" class="btn-close" data-bs-dismiss="alert" aria-label="Fermer"></button>
-        <p class="mb-0">
-            <strong class="text-uppercase">{{ ITOU_ENVIRONMENT }} - Données fictives</strong>
-            {% if not user.is_authenticated %}
-                <i class="ri-arrow-right-line" aria-hidden="true"></i>
-                <button type="button" class="text-white text-decoration-underline" data-bs-toggle="modal" data-bs-target="#testAccountsModal">
-                    Voir les comptes de démonstration
+        <strong class="me-2">{{ ITOU_ENVIRONMENT }} - Données fictives</strong>
+        {% if not user.is_authenticated %}
+            <button type="button" class="btn btn-sm btn-ico btn-outline-white" data-bs-toggle="modal" data-bs-target="#testAccountsModal">
+                <i class="ri-user-community-line" aria-hidden="true"></i>
+                <span>Sélectionner un compte</span>
+            </button>
+        {% else %}
+            <form method="post" action="{% url 'account_logout' %}" class="d-inline">
+                {% csrf_token %}
+                <button type="submit" class="btn btn-sm btn-ico btn-outline-white">
+                    <i class="ri-refresh-line" aria-hidden="true"></i>
+                    <span>Changer de compte</span>
                 </button>
-            {% endif %}
-        </p>
+            </form>
+        {% endif %}
     </div>
 
     <div class="modal fade" id="testAccountsModal" tabindex=-1 role="dialog" aria-modal="true" aria-labelledby="testAccountsModalTitle">
-        <div class="modal-dialog modal-dialog-centered modal-dialog-scrollable modal-xl" role="document">
+        <div class="modal-dialog modal-dialog-centered modal-dialog-scrollable" role="document">
             <div class="modal-content">
                 <div class="modal-header">
-                    <h3 class="modal-title" id="testAccountsModalTitle">Comptes de démonstration</h3>
+                    <h3 class="modal-title" id="testAccountsModalTitle">Sélectionner un compte de démonstration</h3>
                     <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Fermer"></button>
                 </div>
-                <div class="modal-body">
-
+                <div class="modal-body mt-n3">
```

Comment:
```suggestion
                <div class="modal-body mt-3">
```
? (je n'ai pas l'impression que `mt-n3` existe ?)

Reply:
Si mais c'est [confidentiel](https://getbootstrap.com/docs/5.3/utilities/spacing/#negative-margin) :)